dev_guide/cicd.adoc

```diff
@@ -0,0 +1,254 @@
+illustrate how the variety of concepts, infrastructure, hook points, and tools within OpenShift Origin facilitate `*CI/CD*`and the
+promotion of your application through the pipeline, from development, then test, and ultimately to production.
+
+Now, when you consider the following variables:
+
+- whether your team is small or large
+- whether your team is co-located or geographically disperse
+- whether your resources are isolated or shared (or how much isolation or sharing can you afford)
+- whether your processes are fully automated, manual (including the need for human initiated approvals), or some combination of the two
+- whether your processes fall under the waterfall, agile, or devops methodologies, or have elements of any of the three
+
+the matrix that maps scenarios to solutions can seem complex, or at least large.
+
+However, this article will overcome and demystify this.  And it will cherry-pick and consolidate items discussed throughout OpenShift documentation and
+re-present them, but with the specific focus of `*CI/CD*`.
+
+After a brief, high level inventory of components and concepts, followed by a mapping of items either available in OpenShift or easily accessible from OpenShift, to that inventory,
+a series of use cases are provided to illustrate how to best leverage OpenShift Origin in this space.
+
+Mixed into the discussion at various points will be references to the OpenShift Jenkins image as the `*CI/CD*` engine.  With the OpenShift Jenkins Image's accompanying OpenShift templates and
+baked-in set of plugins and configuration that greatly ease the integration between Jenkins and OpenShift combined with the burgeoning list of `*CI/CD*` centric features in
+OpenShift itself, including the Jenkins Pipeline `*BuildConfigStrategy*`, a powerful `*CI/CD*` solution emerges.  That solution includes the ability to auto-provision a Jenkins server to run within
+your OpenShift environment, as well as directly reference and initiate Jenkins Pipeline Jobs.
+
+[[inventory-summary]]
+== Summary of Application Components And Relevant Tools
+
+Spoiler alert:  the "application" is more than just the source code written in say Java, Perl, Python, Ruby, or NodeJS.  It is more now than the static web
+content, the integration scripts, the assoicated configuration for the language specific runtimes for the application, or the application specific archives
+consumed by those language specific runtimes.
+
+By pulling in OpenShift Origin, and its combined foundation of Kubernetes and Docker, additional application artfacts now minimally include:
+- Images with their rich set of metadata and associated tooling
+- Environment Variables and a powerful array of API Objects that are either injected into Containers and Pods for application use, or dictate how the Orchestration Engine (OpenShift on Kubernetes) manages
```

Comment:
list not rendering might be because it needs a blank line between the text and list ... i'll try that and we can see if that helps